src/backend/CryptoDoge.Server/Startup.cs

```diff
@@ -46,6 +178,10 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             app.UseRouting();
 
+            app.UseCors("AllowAny");
```

Comment:
Imho instead of using cors we should set up a proxy for the Frontend dev server.
For example

``` 
         app.UseSpa(spa =>
            {
                spa.Options.SourcePath = "FRONTEND_PATH_HERE";

                if (env.IsDevelopment())
                {
                    spa.UseProxyToSpaDevelopmentServer("http://localhost:4200"); 
                }
            });
```

Reply:
I have never done this before, might worth to take a look. I made the CORS origins configurable for now, so this PR can be merged, because it blocks everything.